website/docs/opengl/1-hello-window.md

```diff
@@ -0,0 +1,263 @@
+---
+{
+    "TableOfContents": {
+        "Name": "1.1 - Hello Window",
+        "Url": "1-hello-window.html",
+        "Metadata": {
+            "AuthorGitHub": "ohtrobinson",
+            "DateTimeWritten": "02/11/2022 12:00",
+            "PreviewImage": ""
+        }
+    }
+}
+---
+
+# 1.1 - Hello Window
+> [!Note]
+> You can view the source code for this tutorial [here.](https://github.com/dotnet/Silk.NET/blob/main/examples/CSharp/OpenGL%20Tutorials/Tutorial%201.1%20-%20Hello%20Window/Program.cs)
```

Comment:
The table looks a bit odd? Maybe if we use more human-readable text, as in instead of "TableOfContents" "Table Of Contents".

I'm not sure about the ```[!Note]``` notation, at least to me it's not displaying correctly.

![image](https://user-images.githubusercontent.com/32400648/199746599-3de3bce0-7761-4809-8428-90f22773ea95.png)

Reply:
This is required for the website to encapsulate metadata and is never seen by the user, see Silk.NET.Statiq. 

As for [!Note], yeah I need to add a shortcode for this at some point to actually turn into a pretty note

Reply:
Ah, I see, forget I said anything lol